duckdb-vortex/src/vortex_scan.cpp

```diff
@@ -159,26 +151,22 @@ static void ExtractVortexSchema(const vx_dtype *file_dtype, vector<LogicalType>
 	}
 }
 
-const  std::regex schema_prefix = std::regex("^[^/]*:\\/\\/.*$");
+const std::regex schema_prefix = std::regex("^[^/]*:\\/\\/.*$");
 
 std::string EnsureFileProtocol(const std::string &path) {
 	// Check if the path has a schema, if not prepend the file:// schema
```

Comment:
I know this was me, but I think the comment needs updating
```suggestion
	// If the path is a URL then don't change it, otherwise try to make the path an absolute path
```